src/components/WalletDropdown/AuthenticatedHeader.tsx

```diff
@@ -139,8 +139,8 @@ const AuthenticatedHeader = () => {
 
   return (
     <AuthenticatedHeaderWrapper>
-      <HeaderWrapper>
-        <StatusWrapper>
+      <HeaderWrapper className="header-wrapper">
+        <StatusWrapper className="something">
```

Comment:
are these classNames necessary?

Reply:
oops my comment didnt get saved earlier - they totally are not necessary lol, thank you for catching